src/quafel/pipelines/visualization/nodes.py

```diff
@@ -28,42 +28,48 @@ class design:
     qual_main = px.colors.qualitative.Dark2  # set1
     qual_second = px.colors.qualitative.Pastel2  # pastel1
 
-    seq_main = px.colors.sequential.Viridis  # pastel1
+    seq_main = px.colors.sequential.thermal  # pastel1
```

Comment:
What does the "pastel1" comment mean?

Reply:
just another option ;)

Reply:
added more descriptive comment, will resolve with #85